src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BotSequence.cs

```diff
@@ -442,6 +451,18 @@ public static (string, string, string) LoadJsonResource(string path)
          */
         public void Play()
         {
+            if (ActiveBotSequence != null)
+            {
+                throw new Exception("Cannot start 2 bot sequences simultaneously.  Another bot sequence is already running.");
+            }
+            ActiveBotSequence = this;
+
+            var playbackController = UnityEngine.Object.FindObjectOfType<BotSegmentsPlaybackController>();
+            if (playbackController.GetState() != PlayState.NotLoaded)
```

Comment:
I'm guessing we don't need to do the null check here because `BotSegmentsPlaybackController` definitely exists if we get to this point?